manifest.json

```diff
@@ -4837,6 +4837,17 @@
                 }
             },
             "versions": {
+            "1.1": {
+                    "releaseUrl": "https://github.com/CatSharkShin/ReArmature/releases/tag/1.1",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/CatSharkShin/ReArmature/releases/download/1.1/ReArmature.dll",
+                            "sha256": "269670969498f187f61ef7b5c343dda0e50d1f28e0eaaf25dc33cfc8f7346434"
+                        }
+                    ]
```

Comment:
you missed a comma after this square bracket, both version blocks need to be indented by 4 more spaces and the versions tag has no closing curly brace